configs/9.0/specs/monolithic.spec

```diff
@@ -191,10 +191,10 @@ systemd_preset=$(pkg-config --variable=systemdsystempresetdir systemd)
 mkdir -p ${RPM_BUILD_ROOT}/${systemd_unit}/
 sed -e s:__AT_VER_REV_INTERNAL__:%{at_ver_rev_internal}: \
     -e s:__AT_DEST__:%{_prefix}: %{_rpmdir}/cachemanager.service \
-    > ${RPM_BUILD_ROOT}/${systemd_unit}/%{at_ver_rev_internal}-cachemanager.service
+    > ${RPM_BUILD_ROOT}/${systemd_unit}/%{at_ver_alternative}-cachemanager.service
 mkdir -p ${RPM_BUILD_ROOT}/${systemd_preset}/
 echo "enable at*cachemanager.service" \
```

Comment:
Although not a blocker, would you also like to change this line so that only the same version gets enabled?